compat/ebt.js

```diff
@@ -7,15 +8,23 @@ exports.init = function (sbot, config) {
   sbot.getAtSequence = ebtIndex.getMessageFromAuthorSequence.bind(ebtIndex)
   sbot.add = sbot.db.add
   sbot.getVectorClock = function (cb) {
-    sbot.db.getAllLatest((err, last) => {
-      if (err) return cb(err)
+    onceWhen(
+      sbot.db2migrate && sbot.db2migrate.synchronized,
```

Comment:
If this first argument in `onceWhen` is falsy, then `onceWhen` will just call the callback. So no worry if you're not using sbot.db2migrate